app/assets/stylesheets/components/cards/prompt.scss

```diff
@@ -0,0 +1,91 @@
+@use "govuk-frontend/dist/govuk" as govuk;
+@use "../../colours";
+
+.prompt {
+  margin: 45px 0 25px;
+  padding-left: 20px;
+  padding-right: 20px;
+  position: relative;
+  background-color: #f3f2f1;
```

Comment:
We should make sure these are variables rather than hardcoded where possible. We're already importing colours here so any custom colours can be moved there